app/javascript/topNavigation/utilities.js

```diff
@@ -53,7 +53,7 @@ export function initializeTouchDevice(memberTopMenu, menuNavButton) {
   }
 
   if (memberTopMenu) {
-    let crayonsHeaderMenuClassList = memberTopMenu.classList;
+    const crayonsHeaderMenuClassList = memberTopMenu.classList;
```

Comment:
@nickytonline Correct me if I'm wrong but IF we're reassigning identifier then shouldn't use `let` instead of `const`, no?

Reply:
`const` and `let` are block scoped (for example stuff in between { and }) which is why you can do this.